cvat/apps/dataset_manager/tests/assets/tasks.json

```diff
@@ -444,5 +444,46 @@
       }
     ]
+  },
+  "YOLOv8 Pose 1.0": {
+    "name": "YOLOv8 pose task",
+    "overlap": 0,
+    "segment_size": 100,
+    "labels": [
+      {
+        "name": "skeleton",
+        "color": "#2080c0",
+        "type": "skeleton",
+        "attributes": [
+          {
+            "name": "attr",
+            "mutable": false,
+            "input_type": "select",
+            "values": ["0", "1", "2"]
+          }
+        ],
+        "sublabels": [
+          {
+            "name": "1",
+            "color": "#d12345",
+            "attributes": [],
+            "type": "points"
+          },
+          {
+            "name": "2",
+            "color": "#350dea",
+            "attributes": [],
+            "type": "points"
+          },
+          {
+            "name": "3",
+            "color": "#479ffe",
+            "attributes": [],
+            "type": "points"
+          }
+        ],
+        "svg": "<line x1=\"38.92810821533203\" y1=\"53.31378173828125\" x2=\"80.23341369628906\" y2=\"18.36313819885254\" stroke=\"black\" data-type=\"edge\" data-node-from=\"2\" stroke-width=\"0.5\" data-node-to=\"3\"></line><line x1=\"30.399484634399414\" y1=\"32.74474334716797\" x2=\"38.92810821533203\" y2=\"53.31378173828125\" stroke=\"black\" data-type=\"edge\" data-node-from=\"1\" stroke-width=\"0.5\" data-node-to=\"2\"></line><circle r=\"1.5\" stroke=\"black\" fill=\"#b3b3b3\" cx=\"30.399484634399414\" cy=\"32.74474334716797\" stroke-width=\"0.1\" data-type=\"element node\" data-element-id=\"1\" data-node-id=\"1\" data-label-name=\"1\"></circle><circle r=\"1.5\" stroke=\"black\" fill=\"#b3b3b3\" cx=\"38.92810821533203\" cy=\"53.31378173828125\" stroke-width=\"0.1\" data-type=\"element node\" data-element-id=\"2\" data-node-id=\"2\" data-label-name=\"2\"></circle><circle r=\"1.5\" stroke=\"black\" fill=\"#b3b3b3\" cx=\"80.23341369628906\" cy=\"18.36313819885254\" stroke-width=\"0.1\" data-type=\"element node\" data-element-id=\"3\" data-node-id=\"3\" data-label-name=\"3\"></circle>"
```

Comment:
It would make for a better test if the circles in the SVG were not in the same order as the sublabels in the `sublabels` field. That way, you could test that the importer is actually taking the order from the SVG.

Reply:
Isn't it the same order?
as far as I understand, sub labels are being loaded here, from svg, and therefore have the same order: https://github.com/cvat-ai/cvat/blob/develop/cvat/apps/dataset_manager/bindings.py#L1465

Reply:
Sure, but the point is to test that nothing unusual happens if the sublabels are listed in a different order in `sublabels` vs `svg`.

Reply:
I don't understand  how that situation can happen

Reply:
The order of the elements in the `sublabels` array and the `svg` string are determined by whoever calls the API. An API client could just list the sublabels in different orders in those two fields.

Reply:
changed the order in `svg`. It caused skeleton annotation elements to change order, so I updated compare_datasets function to allow it.

I am not sure: probably I should update ExactComparator and compare_datasets in datumaro to allow skeleton elements to have different order

Reply:
> I am not sure: probably I should update ExactComparator and compare_datasets in datumaro to allow skeleton elements to have different order

That would probably be a good idea, but I'm okay with the current solution too.

Reply:
added it (and a small bugfix) in https://github.com/cvat-ai/datumaro/pull/57